app/shared-components/PromoModal/PromoModal.jsx

```diff
@@ -23,7 +23,7 @@ import { sendMessage } from '../../panel/utils/msg';
 import globals from '../../../src/classes/Globals';
 import ModalExitButton from '../../panel/components/BuildingBlocks/ModalExitButton';
 
-const { GHOSTERY_ROOT_DOMAIN } = globals;
+const { GHOSTERY_WEBSITE_BASE_URL, CHECKOUT_BASE_URL } = globals;
```

Comment:
Let's use the `GHOSTERY_BASE_URL` value which we already have and which is the same as `GHOSTERY_WEBSITE_BASE_URL`

Reply:
Good catch, I didn't realize we had this already